core/native/src/main/scala/me/shadaj/scalapy/interpreter/Platform.scala

```diff
@@ -57,4 +60,12 @@ object Platform {
   def setPtrLong(ptr: Pointer, offset: Int, value: Long): Unit = !((ptr + offset)).asInstanceOf[Ptr[Long]] = value
   def setPtrInt(ptr: Pointer, offset: Int, value: Int): Unit = !((ptr + offset)).asInstanceOf[Ptr[Int]] = value
   def setPtrByte(ptr: Pointer, offset: Int, value: Byte): Unit = !((ptr + offset)).asInstanceOf[Ptr[Byte]] = value
+
+  def programName: Option[Ptr[CWideChar]] =
```

Comment:
This logic seems duplicated. We should just expose `toCWideString` as a cross-platform API and move the rest of the properties and environment variable logic to shared code.